text/0155-external-modules.md

```diff
@@ -0,0 +1,124 @@
+
+- Create an "experimental" folder within contrib. The contents of this folder are not reviewed or maintained by language repository maintainers, but
+many of the other benefits of being within a contrib repository remain.
+- Add more approvers and maintainers, perhaps some who exclusively focus on submissions to contrib.
+
+### External components
+
+If component authors, for whatever reason, want to host their contribution outside an Otel contrib repository they are free to do so (though we
+encourage all contributions to go into contrib or core language repositories). Their submission for inclusion into OpenTelemetry Registry is still
+welcomed, subject to the same process described above.
+
+### Distribution
+
+Whenever OpenTelemetry components are published to any repository other than the OpenTelemetry Registry (such as npm registry or Maven Central),
+only core and contrib components can be published under "opentelemetry" namespace.
+Native and external components are to be published under their own namespace.
+
+In case the Otel SIG provides any kind of "all-in-one" instrumentation distribution (e.g. as Java and .NET do)
+their should be an option to get a distribution with only core and contrib packages included.
+The OpenTelemetry Registry should provide a way to easily obtain a list of these components.
+The SIG may provide other distributions as well.
+If possible, SIGs should provide a mechanism to include any external component during target application's build- or runtime.
+This may mean a separate language-specific component API that all components are encouraged to implement.
+
+## Trade-offs and mitigations
+
+* How easy it is to get merge permission in contrib repo?
+The harder it is, the larger is the maintenance burden on the core team.
+The easier it is, the more uncertainty there is about the quality of contributions.
+Can every language SIG decide this for themselves or should we decide together?
+
+## Open questions
+
+### Registry self-assessment form
+
+The exact list should be developed separately, but at least the component's author should declare that
+
+* It uses permissive OSS license approved by CNCF
+* It does not have any known security vulnerabilities
+* It produces telemetry which adheres to OpenTelemetry semantic conventions
+* If the Otel/SIG provides a testing harness to verify produced telemetry, that tests were used and passed
+* Authors commit a reasonable effort into future maintenance of this component
+
+### Component information in the Registry
+
+The exact UI of component's page in the Registry is outside of this OTEP's scope, but some suggestions are:
+
+* Link to the original repository of this component
+* Clear indication of component's authors/maintainers and its Otel status (native/core/contrib/external)
+* Licence used
+* Language and targeted library
+* Presence or absence of security vulnerabilities
```

Comment:
maybe just link to the list of known issues? Or this suppose to be provided by a bot?